packages/features/calAIPhone/interfaces/AIPhoneService.interface.ts

```diff
@@ -301,6 +301,22 @@ export interface AIPhoneServiceProvider<T extends AIPhoneServiceProviderType = A
     message: string;
   }>;
 
+  /**
+   * Create a new inbound agent
+   */
+  createInboundAgent(params: {
```

Comment:
Does it mean that when _createAgent_ is called that creates an outbound agent? If yes, we might want to update the description for createAgent and introduce a new _createOutboundAgent_ method that just defers to _createAgent_